src/app/zap-templates/zcl/data-model/chip/application-basic-cluster.xml

```diff
@@ -23,6 +23,7 @@ limitations under the License.
     <define>APPLICATION_BASIC_CLUSTER</define>
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
```

Comment:
What's the purpose of this global attirbute?

Reply:
It's the cluster revision.